settings/drush-request-trace.settings.php

```diff
@@ -19,7 +19,7 @@
   putenv('HTTP_HOST=' . $_SERVER['HTTP_HOST']);
 
   if (function_exists('drush_get_context')) {
-    $cli_args = drush_get_context('argv');
+    $cli_args = $GLOBALS['argv'];
```

Comment:
This is the only deprecated function call we can fix at the moment. I've tested it in the canary application and it's safe.